contracts/Colony.sol

```diff
@@ -28,30 +28,56 @@ contract Colony is ColonyStorage, PatriciaTreeProofs {
   // Version number should be upped with every change in Colony or its dependency contracts or libraries.
   function version() public pure returns (uint256 colonyVersion) { return 1; }
 
-  function setFounderRole(address _user) public stoppable auth {
-    // To allow only one address to have founder role at a time, we have to remove current founder from their role
+  function setRootRole(address _user, bool _setTo) public stoppable auth {
```

Comment:
Do we worry about `DomainRoles` Root role confused with `DSRoles` Root role?

Reply:
Mm I suppose slightly, we could re-use the `Founder` terminology if we prefer.